pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -76,7 +76,10 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 		},
 	}
 	config, _, kvStore := testutils.NewFakeClusterConfigUsingKV(kv)
-	vmiCreateAdmitter := &VMICreateAdmitter{ClusterConfig: config}
```

Comment:
Unrelated to this PR, I find it weird that the setup is not contained in a BeforeEach stanza.
The meaning is that tests are not independent.
I'll create a PR.

Reply:
The bellow test is an example why BeforeEach would not work and require overriding the tested object (i.e.: require injecting crafted network-validator).